doc/source/whatsnew/v0.23.0.txt

```diff
@@ -240,7 +240,7 @@ The :func:`DataFrame.assign` now accepts dependent keyword arguments for python
   using ``.assign()`` to update an existing column. Previously, callables
   referring to other variables being updated would get the "old" values
 
-  Previous Behaviour:
+  Previous behaviour:
```

Comment:
It looks like we generally default to American English spellings within the codebase, so could do "behavior" here instead.  Found many instances of "behaviour" in the codebase though, but still far outnumbered by "behavior".